daemon/oci_linux.go

```diff
@@ -829,15 +829,11 @@ func withCgroups(daemon *Daemon, daemonCfg *dconfig.Config, c *container.Contain
 
 		p := cgroupsPath
 		if useSystemd {
-			initPath, err := cgroups.GetInitCgroup("cpu")
+			path, err := cgroups.GetOwnCgroup("cpu")
```

Comment:
Looks like we don't currently import the `"path"` package, so we're good (I tend to avoid using `path` for variable names nowadays as it's usually waiting for a "I shadowing an import" warnings to happen)